appyx-components/backstack/common/src/commonMain/kotlin/com/bumble/appyx/components/backstack/ui/parallax/BackStackParallax.kt

```diff
@@ -23,6 +26,7 @@ class BackstackParallax<InteractionTarget : Any>(
     defaultAnimationSpec = defaultAnimationSpec,
 ) {
     private val width = uiContext.transitionBounds.widthDp.value
+    private val slowInFastOutEasing = CubicBezierEasing(1f, 0f, 1f, 0f)
```

Comment:
Isn't this easing to dramatic? https://cubic-bezier.com/#1,0,1,0
`FastOutSlowInEasing` doesn't work well here?

Reply:
Yes, that is intentional, the shadow is to be shown until the element is scrolled to the end and then disappear immediately after.